personal_assistant/pa_contacts/templates/pa_contacts/overview.html

```diff
@@ -40,7 +40,14 @@ <h5 class="card-title">
                                         </p>
                                     {% endif %}
                                 </div>
-                                <div class="card-footer">
+                                <div class="card-footer d-flex justify-content-between">
+                                    <a
+                                        href="{% url 'pa_contacts:edit' contact.id %}"
+                                        role="button"
+                                        class="btn btn-primary"
+                                    >
+                                        <i class="bi bi-pencil-fill"></i></i> Edit
```

Comment:
Зайвий закриваючий тег:
```suggestion
                                        <i class="bi bi-pencil-fill"></i> Edit
```